lib/tasks/e2e_seed.rake

```diff
@@ -1,14 +1,21 @@
 # Thanks to https://stackoverflow.com/questions/19872271/adding-a-custom-seed-file/31815032#31815032
 SEED_DIR = Rails.root.join("spec/support/seeds/")
+seed_file = ENV["E2E_SEED_FILE"] || "seeds_e2e.rb"
 
+ENV["E2E_SEED_FILE"]
 namespace :db do
   namespace :seed do
     desc "Seed data for e2e tests"
     task e2e: :environment do
       raise "Attempting to seed production environment, aborting!" if Rails.env.production?
 
-      filename = SEED_DIR.join("seeds_e2e.rb")
-      load(filename) if File.exist?(filename)
+      filename = SEED_DIR.join(seed_file)
+      if File.exist?(filename)
+        load(filename)
+      else
+        raise "Unable to find the seed file #{filename}. If you are passing in a seed file from bin/e2e or bin/e2e-ci,\
+ensure the file exists."
```

Comment:
I tried this but was getting rubocop errors about the line being too long,

```ruby
      raise "Unable to find the seed file #{filename}. If you are passing in a seed file from bin/e2e or bin/e2e-ci, ensure the file exists." unless File.exist?(filename)

      load(filename)
```

so then I tried this

```ruby
      raise "Unable to find the seed file #{filename}. If you are passing in a seed file from bin/e2e or bin/e2e-ci, ens\
ure the file exists." unless File.exist?(filename)

      load(filename)
```

but then Rubocop complained about thid `Favor a normal unless-statement over a modifier clause in a multiline statement.`

In the end what I have in this commit is what made it happy, but just wanted to check in on this @citizen428.

Reply:
@nickytonline What I'd usually do is save the bulk of the long message in a constant with a format string, something like this:

```ruby
SEED_ERROR = "Unable to find the seed file %s<filename>. If you are passing in a seed file from bin/e2e or bin/e2e-ci," \
  "ensure the file exists.".freeze

raise format(SEEED_ERROR, filename: filename) unless File.exist?(filename)
```

It's not strictly necessary, I just find the constant at the top less disruptive for the flow of the code than the long message. When I read I just care that an error is raised and control flow interrupted, not so much about what the exact wording of the error is.

Reply:
I hadn't considered a constant as it's only used in one place, but I like the readability aspect of it.